src/modules/sub-items/components/load-more/load.more.component.js

```diff
@@ -36,25 +32,25 @@ export default class LoadMoreComponent extends Component {
     }
 
     render() {
+        const { totalCount, loadedCount, limit, labels } = this.props;
         const btnAttrs = {
             className: 'c-load-more__btn--load',
             onClick: this.loadMore,
         };
-        const { totalCount, loadedCount, limit } = this.state;
 
         if (!totalCount || loadedCount >= totalCount) {
             btnAttrs.disabled = true;
         }
 
-        const loadMoreText = this.props.labels.info.replace('{{loaded}}', loadedCount).replace('{{total}}', totalCount);
+        const loadMoreText = labels.info.replace('{{loaded}}', loadedCount).replace('{{total}}', totalCount);
         const delta = totalCount - loadedCount;
         const loadMoreLimit = delta > limit ? limit : delta;
-        const actionText = this.props.labels.action.replace('{{limit}}', loadMoreLimit);
+        const actionText = labels.action.replace('{{limit}}', loadMoreLimit);
 
         return (
             <div className="c-load-more">
                 <div className="c-load-more__message" dangerouslySetInnerHTML={{ __html: loadMoreText }} />
-                <div {...btnAttrs} ref={(ref) => (this._refLoadMore = ref)}>
+                <div {...btnAttrs} ref={this.setLoadMoreRef}>
```

Comment:
Do we need `_refLoadMore`? I can see that we use this reference only in one place - in `loadMore` - and only to check if `disabled` flag was set, but this flag is set in the component so in the component we have information if button is disabled thus we don't need reference.

I think that it would be a good idea to extract this: `!totalCount || loadedCount >= totalCount` into another function and use this new function to check this condition in `render` method and in `loadMore` method.

Reply:
This idea is out of scope of this PR.